src/index.js

```diff
@@ -1,9 +1,10 @@
-import footerDom from "./footer";
-import printHeader from "./header";
+import footerDom from './footerjs';
+import printHeader from './header.js';
+import getList from './list.js';
 import './style.css';
```

Comment:
In the import statement, footerjs is a typo, makes npm run build to fail since webpack cannot find a file called footerjs

Reply:
thanks, I have fixed typo